app/models/nta_upload.rb

```diff
@@ -496,15 +506,15 @@ def load_data_postproc
       insert data_list_measurements (data_list_id, measurement_id, indent, list_order)
       select distinct dl.id, m.id, 'indent1',
           case m.data_portal_name
-              when 'Africa' then 1
-              when 'Americas' then 2
-              when 'Asia' then 3
-              when 'Europe' then 4
-              when 'Oceania' then 5
-              when 'Low Income' then 7
-              when 'Lower Middle Income' then 8
-              when 'Upper Middle Income' then 9
-              when 'High Income' then 10
+              when 'African Region' then 1
```

Comment:
The wanted this to be "Countries" not "Region"

Reply:
Well, yeah, but... this is now a Geography, and the "region" series (Asia, Americas, etc) are connected to it, so... it seemed more "logical" :=) to me to word it this way. If you think I should just change it back, it's easy enough.

Reply:
'Course... now that I consider it.. the income pseudo-region groupings are called "blah countries", so I guess it's more consistent that way. Will change.

Reply:
If the naming here does not affect the display on the data portal, feel free to go with your gut.